src/pages/Browse/index.tsx

```diff
@@ -1,20 +1,72 @@
 import * as React from 'react';
 import { RouteComponentProps } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { Col } from 'react-bootstrap';
+
+import { ApplicationState, ConnectedReduxProps } from '../../configureStore';
+import { ApiState } from '../../reducers/api';
+import { callApi } from '../../api';
+import FileTree from '../../components/FileTree';
 
 type PropsFromRouter = {
   versionId: string;
 };
 
-export class BrowseBase extends React.Component<
-  RouteComponentProps<PropsFromRouter>
-> {
+type PropsFromState = {
+  apiState: ApiState;
+};
+
+/* eslint-disable @typescript-eslint/indent */
+type Props = RouteComponentProps<PropsFromRouter> &
+  PropsFromState &
+  ConnectedReduxProps;
+/* eslint-enable @typescript-eslint/indent */
+
+export class BrowseBase extends React.Component<Props> {
+  state = {
+    response: null,
+  };
+
+  async componentDidMount() {
+    const { apiState, match } = this.props;
+    const { versionId } = match.params;
+
+    const response = await callApi({
+      apiState,
+      endpoint: `/reviewers/browse/${versionId}`,
+    });
+
+    this.setState({ response });
+  }
+
   render() {
+    const { response } = this.state;
+
+    // @ts-ignore
+    const showFileTree = response && !response.error;
+
     return (
-      <div>
-        <p>Version ID: {this.props.match.params.versionId}</p>
-      </div>
+      <React.Fragment>
+        <Col md="3">
+          {showFileTree && (
+            <FileTree
+              // @ts-ignore
```

Comment:
Why do we need this? I guess it's just temporary so it's probably okay for now?

Reply:
yeah, it will be removed once your reducer patch lands and we have better types

Reply:
I removed the comments